.github/workflows/pipelines-root.yml

```diff
@@ -147,6 +154,13 @@ jobs:
           PIPELINES_CUSTOMER_ORG_READ_TOKEN: ${{ steps.pipelines-customer-org-read-token.outputs.PIPELINES_TOKEN }}
           PR_COMMENT_WRITE_TOKEN: ${{ steps.pipelines-infra-root-write-token.outputs.PIPELINES_TOKEN }}
 
+      - name: Post Orchestrate Custom Action
+        uses: ./pipelines-actions/.github/custom-actions/post-orchestrate
+        with:
+          PIPELINES_GRUNTWORK_READ_TOKEN: ${{ steps.pipelines-gruntwork-read-token.outputs.PIPELINES_TOKEN }}
```

Comment:
It probably makes sense to pass the orchestrate output into the post action `${{ steps.orchestrate.outputs.jobs }}`